lib/galaxy/datatypes/text.py

```diff
@@ -551,6 +551,53 @@ def set_meta(self, dataset: DatasetProtocol, overwrite: bool = True, **kwd) -> N
                     return
 
 
+@build_sniff_from_prefix
+class CytoscapeJson(Json):
+    """
+    Cytoscape JSON format for network visualization, typically containing 'nodes' and 'edges' in a JSON object.
+    https://js.cytoscape.org/#notation/elements-json
```

Comment:
You can add this link also to the datatypes_conf file as `description` and `description_url`